router/router.go

```diff
@@ -159,7 +171,7 @@ func (r *Router) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}()
 
 	mux := http.NewServeMux()
-	mux.Handle("/query", h)
+	mux.Handle("/", h)
```

Comment:
why the switch from `/query` to `/`?

The goal of `/query` was for future proofing if we wanted to serve other endpoints (e.g. `/ws/services/<id>/stdout`). Still doable with `/` but slightly weird

Reply:
Pushed a commit going back to /query (will squash it on merge).

Reply:
Also opened an issue for the playground here https://github.com/dagger/dagger.io/issues/1341